### DISCUSSION TITLE
Hide breadcrumbs on some pages?

### DISCUSSION THREAD

**Discussion by StarfallProjects** [squidfunk/mkdocs-material#4873] [Q&A] [answered]
Is it possible to hide breadcrumbs on some pages? In the same way you can hide navigation and toc on a page-by-page basis with `hide` in frontmatter.

Or even: a config option so it doesn't show on top-level pages?

**Comment by squidfunk** [comment, marked as answer]
Makes sense! You can raise a [feature request](https://github.com/squidfunk/mkdocs-material/issues/new?assignees=&labels=&template=feature.yml).
